src/psbt.js

```diff
@@ -252,6 +252,57 @@ export function psbtOutputFormatter(output) {
   }
 }
 
+function validatePsbt(network, addressType, psbt) {
+  if (addressType !== P2SH) throw new Error("Unsupported addressType -- only P2SH is supported right now");
+  return autoLoadPSBT(psbt, {network: networkData(network)});
+}
+
+function getInputsFromPsbt(network, addressType, psbt) {
+  let localPSBT = validatePsbt(network, addressType, psbt)
+  if (localPSBT === null) return null;
```

Comment:
Technically these maybe don't need to be here and just in the translate, but my thinking was that in a pure functional world, consumers would use these directly and the object creation that's done in `translate` would be done for each property in the constructors in uc-wallets. Maybe even better would be for these functions to expect an already loaded psbt and not have to load or validate. just brainstorming and clarifying the thinking here.